ci/travis_install_arrow.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2017-2018 Dremio Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+#TAG=apache-arrow-0.9.0
+TAG=master
+ARROW_SRC_DIR=arrow-${TAG}
+
+# Use Ninja for faster builds when using toolchain
+if [ $GANDIVA_TRAVIS_USE_TOOLCHAIN == "1" ]; then
+  CMAKE_ARROW_FLAGS="$CMAKE_COMMON_FLAGS -GNinja"
+fi
+
+wget https://github.com/apache/arrow/archive/${TAG}.zip
```

Comment:
will master be stable enough to take a dependency on?

i would prefer going down the tag route (0.9 and bumping it up to next one after testing etc).

Reply:
I agree. I think we should pick a stable release rather than picking master all the time.

Also, if someone has a dependency on arrow 0.8.0 or some other version, can they make a private copy of gandiva and change the TAG appropriately?

Reply:
I tried with 0.9.0 and 0.8.0 - neither of them works (ldd on the final library still shows dependency on libstdc++/libboost/..). I'm assuming there's some fix that went into arrow codebase after the 0.9.0 release.

Can we track the downgrade to 0.9.0 in a separate jira ?

Reply:
Vivek - this change is primarily for travis and for our packaging a static lib for Dremio.

Direct consumers (who will build from source) are not affected, they can choose to supply the version of arrow they have.

But should we enforce that the arrow being supplied is at-least >= some version?